src/pages/groupComparison/GroupComparisonMutationMapper.tsx

```diff
@@ -46,8 +76,119 @@ export default class GroupComparisonMutationMapper extends MutationMapper<
         return null;
     }
 
+    protected formatPaginationStatusText = (text: string) => {
```

Comment:
this should really be a free function (not a method of class) which accepts text and tableData as arguments.  would be much easier to test.